marklogic-client-api/src/main/java/com/marklogic/client/dataservices/OutputEndpoint.java

```diff
@@ -61,5 +61,9 @@ static OutputEndpoint on(DatabaseClient client, JSONWriteHandle apiDecl) {
          * @param outputConsumer a function for processing the endpoint output
          */
         void forEachOutput(Consumer<InputStream> outputConsumer);
+        /**
+         * Called by the Kafka dataflow connector to get the next batch of output.
```

Comment:
Let's not document the new method as exclusive to Kafka -- it's available for any consumer.  

Instead, would it work to clarify that next() provides synchronous access to output and the output consumer provides asynchronous access to output?

Also, could the JavaDoc document the return value?